gitlab/resource_gitlab_service_slack.go

```diff
@@ -160,49 +183,20 @@ func resourceGitlabServiceSlack() *schema.Resource {
 				Type:        schema.TypeString,
 				Optional:    true,
 			},
-			"job_events": {
```

Comment:
We can't remove job_events from the schema, since it's a breaking change.

Reply:
Hmmm, I agree and I can bring it back if you want. The reason I've removed it is because it was only written to state, but never actually submitted to the upstream API, because [go-gitlab does not support](https://github.com/xanzy/go-gitlab/issues/1354) it.

Should I just add it to the attribute description that it won't work at the moment and leave the attribute in the schema?

Reply:
Wouldn't it break anyone who was setting it in their TF config? If not then it could be fine to remove, but that was my thought.

Yeah if we keep it, please document that it doesn't do anything. 😄

Reply:
Yes, it would :) I brought it back :)